### PR TITLE
@eessex => Deprecate artwork and image sections from article body

### PR DESCRIPTION
**Pull request by kanaabe**
Finally!

### CODE REVIEW

File: api/apps/articles/model/save.coffee [hunk 1]
```diff
@@ -136,7 +136,6 @@ sanitize = (article) ->
   if article.sections
     sections = for section in article.sections
       section.body = sanitizeHtml section.body if section.type is 'text'
-      section.caption = sanitizeHtml section.caption if section.type is 'image'
       section.url = sanitizeLink section.url if section.type is 'video'
```
Comment (kanaabe) [inline api/apps/articles/model/save.coffee]: This reminds me, we should do another audit of new fields (postscript comes to mind) that could be an XSS risk.

File: api/apps/articles/model/schema.coffee [hunk 2]
```diff
@@ -107,7 +107,6 @@ ImageCollectionSection = (->
   gravity_id: @string().objectid().allow('', null)
   hero_section: @alternatives().try(videoSection, ImageCollectionSection, imageSection, featureSection).allow(null).default(null)
   sections: @array().items([
-    imageSection
     ImageCollectionSection
```
Comment (eessex) [inline api/apps/articles/model/schema.coffee]: I think we still need line 110 until the hero_section backfill

File: api/apps/articles/model/save.coffee (hunk 1 above)
Comment (eessex) [reply api/apps/articles/model/save.coffee]: Yes for sure!

**Comment by eessex** [comment]
Hooray :clap: Feel free to merge on 🍏

**Pull request merged**